iterations_runners/iterations_runner.c

```diff
@@ -136,61 +156,40 @@ main(int argc, char **argv)
         }
 
         /* Start timed section */
-        krun_mperf_start = read_mperf();
-        krun_aperf_start = read_aperf();
-        krun_cycles_start = read_core_cycles();
-        krun_wallclock_start = clock_gettime_monotonic();
-
+        krun_measure(0);
```

Comment:
We can yeah, but I don't see it as completely necessary. The reason I used 0 and 1 is because I was not certain all the languages support symbolic constants, so we might still end up with some 0s and 1s for some languages.

FWIW, I don't think of the 0s and 1s as start and stop, but as an array index, hence the parameter name is `mdata_index`.